src/vs/platform/prompts/common/constants.ts

```diff
@@ -33,19 +37,31 @@ export const LOCATIONS_CONFIG_KEY: string = 'chat.promptFilesLocations';
  */
 export const DEFAULT_SOURCE_FOLDER = '.github/prompts';
 
+
+/**
+ * Gets the prompt file type from the provided path.
+ */
+export function getPromptFileType(fileUri: URI): 'instructions' | 'prompt' | undefined {
+	const filename = basename(fileUri);
+	if (filename.endsWith(PROMPT_FILE_EXTENSION)) {
+		return 'prompt';
+	} else if (filename.endsWith(INSTRUCTION_FILE_EXTENSION) || filename === COPILOT_CUSTOM_INSTRUCTIONS_FILENAME) {
+		return 'instructions';
+	}
+	return undefined;
```

Comment:
Nit: the `else` statements are redundant since we return from the if blocks 🤗 I find this style to be bit cleaner:

```suggestion
	if (filename.endsWith(PROMPT_FILE_EXTENSION)) {
		return 'prompt';
	}
	
	if (filename.endsWith(INSTRUCTION_FILE_EXTENSION) || filename === COPILOT_CUSTOM_INSTRUCTIONS_FILENAME) {
		return 'instructions';
	}

	return undefined;
```